doc/api/http2.md

```diff
@@ -1197,8 +1197,11 @@ added: v8.4.0
     **Default:** `false`
   * `parent` {number} Specifies the numeric identifier of a stream the newly
     created stream is dependent on.
-* `callback` {Function} Callback that is called once the push stream has been
-  initiated.
+* `callback` {Function} Callback that is called once the push stream has been  
+initiated.
+  * `err` {Error}
+  * `pushStream` {[`ServerHttp2Stream`][]} The returned pushStream object.
+  * `headers` {[Headers Object][]} Headers object the pushStream was initiated with.
```

Comment:
A nit: this line also needs wrapping (currently 84 characters).

Sorry for the pettiness) This can be fixed by a commit lander if it is not convenient for you to amend the commit or push more commits)

Reply:
it's all good. My first pr, so I'm here to learn.